.gitignore

```diff
@@ -6,6 +6,8 @@ KinSampleApp/KinSampleApp.xcodeproj/project.xcworkspace/xcuserdata
 KinSampleApp/KinSampleApp.xcodeproj/xcuserdata
 
 KinSDK/build
+KinSDK/Geth.framework
```

Comment:
paths are not absolute. add a `/` prefix e.g. `/KinSDK/Geth.framework`
https://git-scm.com/docs/gitignore#_pattern_format

Reply:
also apply this to all paths in this file, regardless of this pr